server/master/src/main/java/org/apache/accumulo/master/Master.java

```diff
@@ -1146,12 +1148,20 @@ private long balanceTablets() {
 
   private SortedMap<TServerInstance,TabletServerStatus> gatherTableInformation(
       Set<TServerInstance> currentServers) {
+    final long rpcTimeout = getConfiguration().getTimeInMillis(Property.GENERAL_RPC_TIMEOUT);
+    int threads = Math.max(getConfiguration().getCount(Property.MASTER_STATUS_THREAD_POOL_SIZE), 0);
```

Comment:
This line seems to imply that negative values will result in unlimited threads. However, the documentation only describes zero threads behaving this way. The docs should say "non-positive" or "zero and negative" just to leave no ambiguity.

Reply:
That was not my intent.  I will removed the max, then the following line would fail if negative.